src/molecules/formfields/SelectField/index.js

```diff
@@ -20,6 +20,11 @@ const renderToolTip = (message) => {
   );
 };
 
+const appliedSelectOptions = ({ selectOptions, value }) => selectOptions.map(option => ({
+  ...option,
+  selected: option.value === value
+}));
+
```

Comment:
Seems like you could just augment the existing functions to take the value prop, instead of passing everything through this pre-function, no? `renderSelectOptions(options, currentValue)`

Reply:
Discussed offline, but that would entail passing the `currentValue` through a bunch of other functions. If this works, i'll clean this logic in a follow up PR.